doc/release-notes/0.15/0.15.md

```diff
@@ -107,7 +107,7 @@ prints statistics about classes on the heap, including number of objects and agg
 <tr><td valign="top"><a href="https://github.com/eclipse-openj9/openj9/pull/2679">#2679</a></td>
 <td valign="top">JEP 331 is implemented</td>
 <td valign="top">OpenJDK11 and later</td>
-<td valign="top">An implementation of <a href="http://openjdk.java.net/jeps/331">JEP 331</a> (Low-Overhead Heap Profiling) is now available. Restrictions are detailed in the <a href="https://eclipse.github.io/openj9-docs/version0.15/">user documentation</a>. </td>
+<td valign="top">An implementation of <a href="http://openjdk.java.net/jeps/331">JEP 331</a> (Low-Overhead Heap Profiling) is now available. Restrictions are detailed in the <a href="https://eclipse-openj9.github.io/openj9-docs/version0.15/">user documentation</a>. </td>
```

Comment:
Pls change this link to the non-draft docs. 
https://www.eclipse.org/openj9/docs/version0.15/

Reply:
Done.